has-guarded-handlers.gemspec

```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
+  s.add_dependency 'thread_safe', ["~> 0.3"]
```

Comment:
when I originally did this 0.3.x was latest - was probably afraid that it might broke around 1.0
but since thread_safe is dead and part of concurrent_ruby gem, latest (last) release is 0.3.5.
so this shouldn't matter much really - but I update to make sure < 0.3.4 is not used - thanks.